evap/staff/tests/test_views.py

```diff
@@ -63,13 +65,17 @@ def test_sample_file_correctness(self):
         page = self.app.get(self.url, user=self.manager)
 
         found_institution_domains = 0
-        book = xlrd.open_workbook(file_contents=page.body)
-        for sheet in book.sheets():
-            for row in sheet.get_rows():
+        book = openpyxl.load_workbook(BytesIO(page.body))
+        for sheet in book:
+            for row in sheet.iter_rows(values_only=True):
                 for cell in row:
-                    value = cell.value
-                    self.assertNotIn(self.email_placeholder, value)
-                    if "@" + settings.INSTITUTION_EMAIL_DOMAINS[0] in value:
+                    # catch None case for empty cells
+                    if not cell:
+                        continue
```

Comment:
implicit bool-conversion will also make empty strings and the number 0 falsy. We only want this to skip None, so it should be `if cell is None`.

I think the comment can be removed.